doc/contributors/contributors.json

```diff
@@ -170,5 +170,9 @@
   {
     "name":"Bruno Casali",
     "link":"https://brunoocasali.xyz"
+  },
+  {
+    "name":"Susan Esho",
+    "link":"https://github.com/andela-oesho"
   }
 ]
```

Comment:
Can you provide a new blank line here?! Git has annoying problems about this... :+1: